stages/org.osbuild.selinux

```diff
@@ -21,7 +21,7 @@ def main(tree, options):
 
     if options.get("force_autorelabel", False):
         stamp = pathlib.Path(tree, ".autorelabel")
-        stamp.touch()
+        stamp.write_text("-F")
```

Comment:
A comment here would be nice, maybe to something like
```
The `.autorelabel` file itself is not enough, it needs options that are read by the `selinux-autorelabel` script [0] so that it properly does a full relabel. Note that this is missing from the selinux(8) and selinux_config(5) man-pages

[0] https://src.fedoraproject.org/rpms/policycoreutils/blob/rawhide/f/selinux-autorelabel#_54 
```
(maybe there is a better link?).

Reply:
I've added a comment, please take a look.